Source/RapyutaSimulationPlugins/Private/Core/RRNetworkPlayerController.cpp

```diff
@@ -226,7 +225,8 @@ void ARRNetworkPlayerController::ServerSetAngularVel_Implementation(ARRBaseRobot
     if (robot)
     {
         float serverCurrentTime = UGameplayStatics::GetRealTimeSeconds(GetWorld());
-        robot->SetActorRotation(InClientRobotRotation + InAngularVel.Rotation() * (serverCurrentTime - InClientTimeStamp));
+        robot->SetActorRotation(InClientRobotRotation +
```

Comment:
what is the difference between 
InAngularVel.Rotation() and FRotator::MakeFromEuler(InAngularVel)

Reply:
So `InAngularVel.Rotation()` calls into [ToOrientationRotator(),](https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Source/Runtime/Core/Private/Math/UnrealMath.cpp#LL199C45-L199C65) but here we would just want the delta rotation I suppose.
Might you help confirm? @larryng-rapyuta